BlotoutAnalytics.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,7 +2,7 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:BlotoutAnalytics.xcodeproj">
+      location = "group:/Users/ankuradhikari/Desktop/a2/Blotoutgithub/ios-sdk/BlotoutAnalyticsSDK.xcodeproj">
```

Comment:
@blotouta2 @nitinblotout we can't have user specific location here

Reply:
We can git ignore this file.

Reply:
Reverted.Done.